### ISSUE TITLE
Is it possible to provision users to SAP Netweaver from Cloud IAS ?

### ISSUE THREAD

**Issue by lmvdz** [agiguere/ZIAS_SCIM_REST_API#1]
Hi I wanted to know if it was possible to use this ABAP to connect SAP Cloud IAS to my SAP NetWeaver system?

**Comment by agiguere** [comment]
absolutely, SAP Cloud IAS offer a REST API that you can use to create users & groups, instead of paying for a product like SAP Identity Provisioning, you can use this open source project and create your users as you would from any SAP Netweaver systems like SAP ECC or SAP CUA ...
